components/raftstore/src/store/worker/split_controller.rs

```diff
@@ -55,31 +55,51 @@ where
     .collect()
 }
 
+#[inline(always)]
+fn prefix_sum_mut<F, T>(iter: IterMut<'_, T>, read: F) -> Vec<usize>
```

Comment:
Do we need to remove function `prefix_sum`?

Reply:
No. `prefix_sum` is still used elsewhere.